### PR TITLE
Ubuntu 16.04

### PR DESCRIPTION
**Pull request by jburel**
Add ubuntu 16.04 installation page

cc @hflynn

### CODE REVIEW

File: omero/sysadmins/unix/server-ubuntu1604-ice36.txt [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+-----------------------
+
+OMERO.web is deployed using Nginx see :doc:`install-web/install-nginx`.
+
+**The following steps are run as root.**
+
+Install Nginx 1.8, copy the Nginx OMERO configuration file into the
+Nginx configuration directory, and disable the default configuration:
+
+.. literalinclude:: walkthrough/walkthrough_ubuntu1404.sh
```
Comment (mtbc) [inline omero/sysadmins/unix/server-ubuntu1604-ice36.txt]: It's okay that we use the 14.04 one here?

Comment (jburel) [reply omero/sysadmins/unix/server-ubuntu1604-ice36.txt]: well spotted

**Comment by hflynn** [comment]
I'm completely failing to see any differences between this walkthrough and the one for Ubuntu 14.04 -  do we expect them to diverge or could we have one walkthrough for both versions?

**Comment by hflynn** [comment]
If we are keeping a separate walkthrough, a link also needs to be added to the top of https://www.openmicroscopy.org/site/support/omero5.3-staging/sysadmins/unix/server-installation.html

**Comment by jburel** [comment]
There is a small diff in the postgres installation
``apt-get -y install postgresql`` will install 9.3 on ubuntu 14.04

maybe we can combine the 2

**Comment by hflynn** [comment]
Ah, I see! Yes, i think we could combine them if the postgres install is the only difference.

**Comment by jburel** [comment]
Closing this PR, will combine the ubuntu installation once gh-1614 is merged

**Pull request closed without merging**